src/js/components/Books/Books.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import Navbar from '../Navbar/Navbar';
 import Bookcard from '../Books/Bookcard/Bookcard';
 
-import './Book.sass';
+import './Book.scss';
 class  Books extends Component {
   render() {
     var loc_navBarTitle  = "BOOKS";
```

Comment:
Use let and keep your naming consistent

Reply:
Thank  you for that.

Reply:
It was because the file had been changed.

Reply:
You haven't changed the var variable to let

Reply:
Thank you I have changed it to let